python/setup.py

```diff
@@ -45,7 +45,9 @@ def get_lib_path():
         'numpy>=1.14.0',
         'scipy>=1.1.0',
         'networkx>=2.1',
-        'hyperopt'
+        'hyperopt',
+        'dgl',
```

Comment:
I don't recommend putting "dgl" here. The reason is that there are different variants of "dgl" for CPU/CUDA 9.0/CUDA 10.0, etc and it's better to let people install the version they want themselves.